### PR TITLE
Add setting to not aggregate reblogs

### LINKED ISSUE #9222: Add an option to opt-out from boosts aggregation
### Pitch

The idea is to have an option to see all individual boosts from your followers and not just the first boost to appear on your timeline on your current session. This option would not be activated by default of course.

### Motivation

Some people prefer to know who boosted a toot and which toots are more important than others (i.e. with this feature such a toot would appear multiple times on your timeline) even if this means getting a lot more activity on your timeline and getting redundant posts.

### PR DESCRIPTION
**Pull request by ClearlyClaire**
Fixes #9222

### CODE REVIEW

File: app/lib/feed_manager.rb [hunk 1]
```diff
@@ -226,11 +226,11 @@ def phrase_filtered?(status, receiver_id, context)
   # added, and false if it was not added to the feed. Note that this is
   # an internal helper: callers must call trim or push updates if
   # either action is appropriate.
-  def add_to_feed(timeline_type, account_id, status)
+  def add_to_feed(timeline_type, account_id, status, aggregate_reblogs = true)
     timeline_key = key(timeline_type, account_id)
     reblog_key   = key(timeline_type, account_id, 'reblogs')
 
-    if status.reblog?
+    if status.reblog? && (aggregate_reblogs.nil? || aggregate_reblogs)
```
Comment (ThibG) [inline app/lib/feed_manager.rb]: This is a bit awkward, but basically none of the tests pass an actual local account (with an associated `User`) to those functions, so…

**Comment by kit-ty-kate** [comment]
@Gargron any opinion on this?

**Comment by kit-ty-kate** [comment]
FYI: I'm using this PR on top of `2.6.2` on my instance [lesbiab.space](https://lesbiab.space) and it works perfectly so far

**Comment by kedamaDQ** [comment]
I often see the actions which are boost and mention. But boosted toot is not visible in my home timeline and only mentioning is visible.
I think to this is a very good function if frequently need to open the account timeline for check to "what is mentioned?".

Otherwise,  behavior remains as it is unless actively opt-outed, I like it.

**Comment by mayaeh** [comment]
I think this is useful.

In Japanese, especially in my visible range, I often see that they boosting post and then post impressions ( eg: I like this).
Then I will search for what he is talking about.

This PR can omit this procedure.

However, there are times when I do not want to see the same post many times.

Also, it may be better to show that this setting does not affect posts received before changing this setting.

Also, since the space between the hint and the next setting item is too narrow, I think that adjustment is necessary.

**Comment by ThibG** [comment]
Updated the spacing and hint text

screenshot_2018-12-05 preferences - dev instance 1

**Review by mayaeh — APPROVED** [review]

**Comment by kedamaDQ** [comment]
thank you! thank you for merging!!

**Pull request merged**